WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -250,6 +256,21 @@ final class DashboardViewModel: ObservableObject {
 
         analytics.track(event: .Dashboard.dashboardTimezonesDiffers(localTimezone: localGMTOffsetInHours, storeTimezone: siteGMTOffset))
     }
+
+    func saveJetpackBenefitBannerDismissedTime() {
+        let dismissAction = AppSettingsAction.setJetpackBenefitsBannerLastDismissedTime(time: Date())
+        stores.dispatch(dismissAction)
+    }
+
+    func maybeSyncAnnouncementsAfterWebViewDismissed() {
+        // If the web view was opened from a modal JITM, it was dismissed before the webview
+        // was presented. Syncing in that situation would result in it showing again.
```

Comment:
If I understand this correctly,
```suggestion
        // was presented. Syncing in that situation would result in the JITM model showing again.
```

Reply:
I copied the whole method as-is from the old `DashboardViewController`. Updated the wording in 8141bef2f6e9a0591cb051b784bdd8ea47561aa6, I hope that's correct.